apps/amplify-demo/README.md

```diff
@@ -79,29 +87,68 @@ Run the command below and answer the prompts as shown below::
+
+GraphQL schema compiled successfully.
+
+Edit your schema at /<path-to-amplify-demo>/amplify/backend/api/reactSampleApi/schema.graphql or place .graphql files in a directory at /<path-to-amplify-demo>/amplify/backend/api/reactSampleApi/schema
+✔ Successfully pulled backend environment dev from the cloud.
+⠹ Building resource api/reactSampleApi
+⚠️  WARNING: your GraphQL API currently allows public create, read, update, and delete access to all models via an API Key. To configure PRODUCTION-READY authorization rules, review: https://docs.amplify.aws/cli/graphql/authorization-rules
+
+GraphQL schema compiled successfully.
+
+Edit your schema at /<path-to-amplify-demo>/amplify/backend/api/reactSampleApi/schema.graphql or place .graphql files in a directory at /<path-to-amplify-demo>/amplify/backend/api/reactSampleApi/schema
+
+    Current Environment: dev
+    
+┌──────────┬───────────────────┬───────────┬───────────────────┐
+│ Category │ Resource name     │ Operation │ Provider plugin   │
+├──────────┼───────────────────┼───────────┼───────────────────┤
+│ Function │ reactSampleLambda │ Create    │ awscloudformation │
+├──────────┼───────────────────┼───────────┼───────────────────┤
+│ Api      │ reactSampleApi    │ Create    │ awscloudformation │
+└──────────┴───────────────────┴───────────┴───────────────────┘
+? Are you sure you want to continue? (Y/n) y
+.
+.
+.
+? Do you want to generate code for your newly created GraphQL API Yes
+? Choose the code generation language target typescript
+? Enter the file name pattern of graphql queries, mutations and subscriptions src/graphql/**/*.ts
+? Do you want to generate/update all possible GraphQL operations - queries, mutations and subscriptions Yes
+? Enter maximum statement depth [increase from default if your schema is deeply nested] 2
+? Enter the file name for the generated code src/API.ts
   ```
 
+  This will end with success output providing generated GraphQL endpoint and API KEY.
+
 #### 4. Modify IAM Role policy to include Amazon Chime Full Access in order to allow your Lambda function to call Amazon Chime APIs, such as createMeeting and deleteMeeting:
-    1. Navigate to the AWS Console using the same account that you used to configure amplify. 
-    2. Navigate to IAM
-    3. Click on Roles in the side menu. 
-    4. Find the role created and click on the name of the Role -  reactSampleLambdaRole-XXX-{environment}
+    1. Goto your account under AWS CloudFormation stacks click "amplify-<your-demo-name>-dev-*".
+    2. Under "Resources" click on the Physical ID link against "functionreactSampleLambda" (function<name provided when you executed amplify add function>).
+    3. A nested stack will open and click again "Resources".
+    4. Click Physical ID link in front of "LambdaExecutionRole". This will take you to the lambda execution role in IAM.
```

Comment:
nit: `Click` => `Click on` to match other items. (In AWS docs, they often use `Choose` instead of `Click` or `Click on`.)

Reply:
Used "choose" over "click on" or "click".